cimr/prompt.py

```diff
@@ -163,18 +163,36 @@ def add_subparser_processor(subparsers):
     
     # integrate-specific arguments
     parser.add_argument(
-        '--can-be-public',
+        '-can-be-public',
         default=True,
         dest='can_be_public',
         help='a boolean variable indicating whether the integrated data can '
              'be made public. default is True.',
     )
     parser.add_argument(
-        '--temp-dir',
+        '-temp-dir',
         default='cimr-adb-temp',
         dest='temp_dir',
         help='temporary directory name to clone cimr database into.',
     )
+    parser.add_argument(
+        '-study_id',
+        dest='study_id',
```

Comment:
You can force its type to be `int`, if it is helpful.

Reply:
I am still trying to decide whether to require one type here... but in whichever case, this will be str.